package.json

```diff
@@ -10,7 +10,7 @@
     ],
     "bin" : { "forcedroid" : "node/forcedroid.js" },
     "dependencies": {
-        "shelljs": ">=0.1.4"
+        "shelljs": "~0.5.3"
```

Comment:
~a.b.c means accept a.b.x where x >= c
according to http://stackoverflow.com/questions/22343224/difference-between-tilde-and-caret-in-package-json
So 0.6.0 won't be used